app/controllers/sessions_controller.rb

```diff
@@ -49,6 +58,12 @@ def ask_new_or_returning
   def i_am_returning
   end
 
+  # This is an official action instead of just doing `redirect_back` in callback
+  # handler so that fine_print can check to see if terms need to be signed.
```

Comment:
This is needed when someone is just signing in directly to Accounts (without getting there from an app like Tutor) -- i.e. when there's no doorkeeper authorize route in the mix.  Discovered this when writing the last scenario in the feature spec below.